.github/workflows/canary-integration-test.yml

```diff
@@ -1658,19 +1658,7 @@ jobs:
         uses: ./.github/workflows/canary-test-config
 
       - name: set Ceph version in CephCluster manifest
-        run: tests/scripts/github-action-helper.sh replace_ceph_image  "deploy/examples/cluster-test.yaml" "${{ github.event.inputs.ceph-image }}"
-
-      - name: install deps
```

Comment:
Good catch, I guess all these removed steps are already in the canary-test-config